pkg/ddl/ingest/env.go

```diff
@@ -108,5 +110,82 @@ func genLightningDataDir() (string, error) {
 	return sortPath, nil
 }
 
-// GenLightningDataDirForTest is only used for test.
-var GenLightningDataDirForTest = genLightningDataDir
+// CleanUpTempDir reads the temp dir and removes the stale index data.
+// This function will get
```

Comment:
comment is not completed

Reply:
https://github.com/pingcap/tidb/pull/54008/commits/2a3823e2ba1edb02981b313ce5ea1ad78437d694